lib/permutat.g

```diff
@@ -722,6 +722,7 @@ InstallMethod( Order,
     [ IsPerm ],
     ORDER_PERM );
 
+
 #############################################################################
```

Comment:
This file is included in this PR, but only because an empty line has been added? Could you revert it, please?